lib/code-caller/code-caller-container.js

```diff
@@ -453,7 +461,7 @@ class CodeCallerContainer {
 
   _clearTimeout() {
     this.debug('enter _clearTimeout()');
-    clearTimeout(this.timeoutID);
+    clearTimeout(this.timeoutID ?? undefined);
```

Comment:
Could we avoid this change by setting timeoutID to `undefined` rather than `null` when we clear it?

Reply:
Yep; we could also just `if (this.timeoutID) { ... }`, which might be better? I do think `null` is a better "no value" sentinel than `undefined` in most cases.

Reply:
I agree that I prefer `null` as a sentinel. I'm fine with the `?? undefined`, I was just checking to make sure I was understanding why we were doing it.